src/elements/app-bar/brand.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+
+import { classNames } from 'utils'
+
+const Brand = ({ children, className, tag: Tag = 'a', ...restProps }) => (
```

Comment:
I was expecting to have logo here too.

Reply:
Ohh sorry I missed this one https://github.com/oacore/design/issues/49

Reply:
It's not expected. You can pass whatever as a children but the logo is not this component's responsibility.